tests/plugins/balloontoolbar/context/manual/widgetdnd.html

```diff
@@ -0,0 +1,70 @@
+<textarea id="editor1" cols="10" rows="10">
```

Comment:
Please add styles from `contents.css` also to inline editor to have sensible styles for `figure`.